mod_reforged/hooks/skills/effects/berserker_rage_effect.nut

```diff
@@ -0,0 +1,46 @@
+::mods_hookExactClass("skills/effects/berserker_rage_effect", function (o) {
+	local create = o.create;
+	o.create = function()
+	{
+		create();
+        this.m.Description = "This character is in a berserking rage."
+	}
+
+	o.getTooltip <- function()
+	{
+		local ret = this.skill.getTooltip();
+		ret.extend([
+			{
+				id = 10,
+				type = "text",
+                icon = "ui/icons/special.png",
+				text = "Receives only " + ::MSU.Text.colorGreen((this.calcDamageTakenMult() * 100.0) + "%") + " of any damage"
+			},
+			{
+				id = 11,
+				type = "text",
+				icon = "ui/icons/regular_damage.png",
+				text = ::MSU.Text.colorGreen("+" + this.m.RageStacks + "%") + " Damage"
```

Comment:
The damage increase given by this effect is additive, not a percentage. So this should be `+x Damage`